ui/app/templates/vault/cluster/access/methods.hbs

```diff
@@ -15,137 +15,74 @@
 </Toolbar>
 
 {{#each (sort-by "path" model) as |method|}}
-  {{#if (eq method.methodType "ldap")}}
-    <LinkedBlock @params={{array
-    "vault.cluster.access.method"
-    method.id}} class="list-item-row" data-test-auth-backend-link={{method.id}}>
-      <div class="level is-mobile">
-        <div class="level-left">
-          <div>
-            <ToolTip @horizontalPosition="left" as |T|>
-              <T.trigger>
-                <Icon @glyph={{if
-                  (or
-                    (find-by "type" method.methodType (mountable-auth-methods))
-                    (eq method.methodType "token")
-                  )
-                  method.methodType
-                  "auth"
-                }} @size="l" class="has-text-grey-light" />
-              </T.trigger>
-              <T.content @class="tool-tip">
-                <div class="box">
-                  {{method.methodType}}
-                </div>
-              </T.content>
-            </ToolTip>
-            <span data-test-path data-test-id={{method.id}} class="has-text-weight-semibold has-text-black">
-              {{method.path}}
-            </span>
-            <br />
-            <code class="has-text-grey is-size-8">
-            {{method.accessor}}
-            </code>
-          </div>
-        </div>
-        <div class="level-right is-flex is-paddingless is-marginless">
-          <div class="level-item">
-            {{#popup-menu name="auth-backend-nav"}}
-              <nav class="menu">
-                <ul class="menu-list">
-                  <li>
-                    {{#link-to "vault.cluster.access.method.section" method.id "configuration"}}
-                      View configuration
-                    {{/link-to}}
-                  </li>
-                  {{#if method.canEdit}}
-                    <li>
-                      {{#link-to "vault.cluster.settings.auth.configure" method.id}}
-                        Edit configuration
-                      {{/link-to}}
-                    </li>
-                  {{/if}}
-
-                  {{#if (and (not-eq method.methodType 'token') method.canDisable)}}
-                    <li class="action">
-                      <ConfirmAction @buttonClasses="link is-destroy" @confirmTitle="Disable method?"
-                        @confirmMessage="This may affect access to Vault data." @confirmButtonText="Disable"
-                        @onConfirmAction={{perform disableMethod method}}>
-                        Disable
-                      </ConfirmAction>
-                    </li>
-                  {{/if}}
-                </ul>
-              </nav>
-            {{/popup-menu}}
-          </div>
+  <LinkedBlock @params={{array
```

Comment:
Won't there still be some auth methods that don't have links? If so, I think we may need a helper or sort into supported with links up top and then unsupported without links at the bottom - the secret engines list page does that.

Reply:
oh, i'm not sure about this -- why would some auth methods never have links?

Reply:
We're only adding LDAP + the ones listed in the PR description, so anything not in that list shouldn't be links because we don't know that they'll generate the UI correctly from the Open API endpoint.